src/ResourceManager/StorageSync/storageSyncDevops.psm1

```diff
@@ -0,0 +1,186 @@
+function Get-RepositoryRootDirectory
```

Comment:
@afedyashov what is the purpose of this script?

Reply:
devops utils module for our StorageSync project.